main.py

```diff
@@ -1,30 +1,22 @@
-import getpass
+#!/usr/bin/env python3
+
 from collections import namedtuple
-from lxml import objectify
+import os.path
 from project import Project
 from importer import Importer
 from labelcolourselector import LabelColourSelector
-
-
-def read_xml_sourcefile(file_names):
-    files = list()
-    for file_name in file_names.split(';'):
-        all_text = open(file_name).read()
-        files.append(objectify.fromstring(all_text))
-
-    return files
-
-
-file_names = input(
-    'Path to JIRA XML query file (semi-colon separate for multiple files): ')
-all_xml_files = read_xml_sourcefile(file_names)
-
-jira_proj = input('JIRA project name: ') or 'INFRA'
-jira_done_id = input('JIRA Done statusCategory ID [default "3"]: ') or '3'
-jira_base_url = input('JIRA base url [default "https://issues.jenkins.io"]: ') or 'https://issues.jenkins.io'
-ac = input('GitHub account name (user/org): ') or 'jenkins-infra'
-repo = input('GitHub repository name: ') or 'helpdesk'
-pat = input('Github Personal Access Token: ') # or '<your-github-pat>'
+from utils import read_xml_files
+
+file_names = os.getenv('JIRA_MIGRATION_FILE_PATHS') or input(
```

Comment:
very annoying running this over and over again with prompts, I changed it to default to env variables